recipes/myst-nb/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "myst-nb" %}
+{% set version = "0.10.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/myst-nb-{{ version }}.tar.gz
+  sha256: 359c9a200706c80d73f47dce4c986681947952f91883c195ae58a24855075b0f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - docutils >=0.15
+    - importlib-metadata
+    - ipython
+    - ipywidgets >=7.0.0,<8
+    - jupyter-cache ~=0.4.1
+    - jupyter-sphinx ~=0.3.1
+    - myst-parser ~=0.12.5
+    - nbconvert ~=5.6
+    - nbformat ~=5.0
+    - python >=3.6
+    - pyyaml
+    - sphinx >=2,<4
+    - sphinx-togglebutton ~=0.2.2
+
+test:
+  imports:
+    - myst_nb
+    - myst_nb.nb_glue
+
+about:
+  home: https://github.com/executablebooks/myst_nb
+  summary: A Jupyter Notebook Sphinx reader built on top of the MyST markdown parser.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - anirrudh
+    - chrisjsewell
```

Comment:
@chrisjsewell please confirm that you are willing to be a maintainer (so there is a record here)

Reply:
Can we just close this and use my PR.

Reply:
Which ever one works first, I'll merge

Reply:
They will both work at the same time, because they are both waiting for jupyter-cache to be updated